### PR TITLE
Space samurai armour

### PR DESCRIPTION
**Pull request by bombasticSlacks**
#### Summary
None

#### Purpose of change
For the weaver fight

#### Describe the solution
added new armour, some enchantments, more description soon.

#### Additional context
image

### CODE REVIEW

File: data/json/items/armor/exotic.json [hunk 1]
```diff
@@ -252,5 +252,206 @@
...
+    ],
+    "armor": [ { "encumbrance": 2, "coverage": 100, "covers": [ "torso" ], "specifically_covers": [ "torso_waist" ] } ],
+    "relic_data": { "passive_effects": [ { "id": "ench_meteor_armor" } ] }
+  },
+  {
+    "id": "armor_star_samurai_mask",
+    "type": "TOOL_ARMOR",
+    "name": { "str": "filtering menpo" },
+    "category": "armor",
+    "description": "A sealed mask with strange filtering components on the front turned up in the shape of horns.  Inscryptions mark the glass of the lenses creating spiral patterns that blur the vision.",
```
Comment (cosmo-naut) [inline data/json/items/armor/exotic.json:353]: Inscryptions -> Inscriptions

File: doc/MAGIC.md [hunk 2]
```diff
@@ -810,6 +810,8 @@ Character status value  | Description
 `FOOTSTEP_NOISE`        | 
 `FORCEFIELD`            | Chance your character reduces incoming damage to 0. From 0.0 (no chance), to 1.0 (100% chance to avoid attacks).
 `HUNGER`                | 
+`KNOCKBACK_RESIST`      | The ammount knockback effects you, 0 is the regular ammount, -100 would be double effect, 100 would be no effect
```
Comment (cosmo-naut) [inline doc/MAGIC.md:813]: ammount -> amount

File: doc/MAGIC.md [hunk 3]
```diff
@@ -810,6 +810,8 @@ Character status value  | Description
...
+`KNOCKDOWN_RESIST`      | The ammount knockdown effects you, currently *only* having 100 or greater knockdown_resist makes you immune to knockdown
```
Comment (cosmo-naut) [inline doc/MAGIC.md:814]: ammount -> amount

**Review by cosmo-naut — COMMENTED** [review]
minor spelling

**Review by TheShadowFerret — COMMENTED** [review]
~~Where's my STAR_FINGER flag~~
image

File: data/json/items/armor/exotic.json [hunk 4]
```diff
@@ -252,5 +252,212 @@
...
+        "weight_multiplier": 0.0
+      }
+    ],
+    "armor": [ { "encumbrance": 2, "coverage": 100, "covers": [ "torso" ], "specifically_covers": [ "torso_waist" ] } ],
+    "relic_data": { "passive_effects": [ { "id": "ench_meteor_armor" } ] }
+  },
+  {
+    "id": "armor_star_samurai_mask",
+    "type": "TOOL_ARMOR",
+    "name": { "str": "filtering menpo" },
```
[the remark this answers is not in the document]
Comment (bombasticSlacks) [reply data/json/items/armor/exotic.json:357]: That's what I thought, I missed this one though TY!

File: data/json/items/resources/misc.json [hunk 5]
```diff
@@ -280,5 +280,19 @@
     "material": [ "rubber", "steel" ],
     "symbol": "o",
     "color": "dark_gray"
+  },
+  {
+    "id": "armor_star_samurai_helmet_destroyed",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": { "str": "destroyed ryūsei kabuto" },
```
Comment (Qrox) [inline data/json/items/resources/misc.json:288]:
```suggestion
    "name": { "str_sp": "destroyed ryūsei kabuto" },
```

**Pull request merged**